CHANGELOG.md

```diff
@@ -1,22 +1,21 @@
-<!--
-SPDX-FileCopyrightText: 2020 Helmholtz Centre for Environmental Research (UFZ)
-SPDX-FileCopyrightText: 2020 Helmholtz-Zentrum Dresden-Rossendorf (HZDR)
+# Changelog
 
-SPDX-License-Identifier: Apache-2.0
--->
+## [Unreleased](https://github.com/hifis-net/ssh-keys/tree/HEAD)
 
-# Changelog
+[Full Changelog](https://github.com/hifis-net/ssh-keys/compare/v1.2.0...HEAD)
 
-All notable changes to this project will be documented in this file.
+**Merged pull requests:**
 
-The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
-and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+- Implement GitHub actions workflows [\#2](https://github.com/hifis-net/ssh-keys/pull/2) ([Normo](https://github.com/Normo))
 
-Group your changes into these categories:
+<!--
+SPDX-FileCopyrightText: 2020 Helmholtz Centre for Environmental Research (UFZ)
+SPDX-FileCopyrightText: 2020 Helmholtz-Zentrum Dresden-Rossendorf (HZDR)
 
-`Added`, `Changed`, `Deprecated`, `Removed`, `Fixed`, `Security`.
+SPDX-License-Identifier: Apache-2.0
+-->
 
-## [1.2.0](https://gitlab.com/hifis/ansible/ssh-keys/-/releases/v1.2.0) - 2021-02-08
+## [v1.2.0](https://github.com/hifis-net/ssh-keys/releases/tag/v1.2.0) - 2021-02-08
 
 [List of commits](https://gitlab.com/hifis/ansible/ssh-keys/-/compare/v1.1.0...v1.2.0)
```

Comment:
Dear Norman @Normo, I am wondering whether we need to change these links for the list of commits to the one on GitHub as well? But we can't because it is auto-generated, right? And I guess it is on purpose to leave it like that?

Reply:
@christianhueserhzdr Could you please check again?